app/views/notification_mailer/ticket_assigned.text.erb

```diff
@@ -5,7 +5,7 @@
 <%= t('notify_ticket_assigned', default: 'You have recently been assigned to the following topic(s):') %>
 
 <% @bulk_topics.each do |topic| %>
-  <%= topic.name %>
+  <%= topic.name + '  '%><%= link_to t('view_on_site', default: "View on Site"), topic %>
```

Comment:
Yeah that seems reasonable.

The only problem I can see is that what happens if someone does try to reply from the email...

Is there any way to ignore a reply if it is a "bulk" notification? Maybe @scott knows a way?